### PR TITLE
Added Flight Authorization Data to SCD testing API

### PR DESCRIPTION
**Pull request by hrishiballal**
This PR adds to the SCD testing API by adding flight authorization data

### CODE REVIEW

File: interfaces/automated-testing/scd/scd.yaml [hunk 1]
```diff
@@ -36,6 +36,510 @@ components:
...
+            If an existing subscription is not specified in `subscription_id`,
+            and the operational intent is in the Activated, Nonconforming, or
+            Contingent state, then this field must be populated.  When this
+            field is populated, an implicit subscription will be created and
+            associated with this operational intent, and will generally be
+            deleted automatically upon the deletion of this operational intent.
+          anyOf:
+          - $ref: '#/components/schemas/ImplicitSubscriptionParameters'
+
+    FlightAutorizationData:
```
Comment (BenjaminPelletier) [inline interfaces/automated-testing/scd/scd.yaml:450]: Typo: missing an 'r'

Comment (BenjaminPelletier) [reply interfaces/automated-testing/scd/scd.yaml:450]: nit: Given that this is purely to fulfill the European U-space regulation, it would probably be appropriate to adopt the European spelling of "authorisation"

File: interfaces/automated-testing/scd/scd.yaml [hunk 2]
```diff
@@ -36,6 +36,510 @@ components:
...
+      description: |-
+        A dataset to hold deatils of a UAS flight authorization request matching some of the requirements of Annex IV of the U-Space regulation 2021/664
```
Comment (BenjaminPelletier) [inline interfaces/automated-testing/scd/scd.yaml:452]: Typo: "details"

File: interfaces/automated-testing/scd/scd.yaml [hunk 3]
```diff
@@ -36,6 +36,510 @@ components:
...
+      type: object
+      properties:
+        uas_serial_number:
+          type: string
+          example: "T4S6BBMSZV4H0650"
+          description: Serial number of the UAS per the CTA-2063-A standard
+        operation_mode:
```
Comment (BenjaminPelletier) [inline interfaces/automated-testing/scd/scd.yaml:459]: This type is fully defined below and should not inherit from the separate type.  Declaration should simply be:

```yaml
operation_mode:
  $ref: '#/components/schemas/OperationMode'
```

File: interfaces/automated-testing/scd/scd.yaml [hunk 4]
```diff
@@ -36,6 +36,510 @@ components:
...
+          description: Serial number of the UAS per the CTA-2063-A standard
+        operation_mode:   
+          type: string      
+          anyOf:
+            - $ref: '#/components/schemas/OperationMode'
+        operation_category:
+          type: string
+          example: "normal"
+          description: |-
+            Specify if the flight is "special" or has a specific priority, please refer the appropriate regulation for specifiying priority.
```
Comment (BenjaminPelletier) [inline interfaces/automated-testing/scd/scd.yaml:467]: No need to tell the user to refer to the regulation; they should be doing that for all of these fields.

File: interfaces/automated-testing/scd/scd.yaml [hunk 5]
```diff
@@ -36,6 +36,510 @@ components:
...
+        uas_class:
```
Comment (BenjaminPelletier) [inline interfaces/automated-testing/scd/scd.yaml:468]: Same comment as for `operation_mode`

File: interfaces/automated-testing/scd/scd.yaml [hunk 6]
```diff
@@ -36,6 +36,510 @@ components:
...
+        operation_category:
+          type: string
+          example: "normal"
+          description: |-
+            Specify if the flight is "special" or has a specific priority, please refer the appropriate regulation for specifiying priority.
+        uas_class: 
+          type: string
+          anyOf:
+            - $ref: '#/components/schemas/UASClass'
+        identification_technologies:
```
Comment (BenjaminPelletier) [inline interfaces/automated-testing/scd/scd.yaml:472]: Same comment as for `operation_mode`, but also this field is plural while the data type is singular.  We probably want an array here.

File: interfaces/automated-testing/scd/scd.yaml [hunk 7]
```diff
@@ -36,6 +36,510 @@ components:
...
+            Specify if the flight is "special" or has a specific priority, please refer the appropriate regulation for specifiying priority.
+        uas_class: 
+          type: string
+          anyOf:
+            - $ref: '#/components/schemas/UASClass'
+        identification_technologies: 
+          type: string
+          anyOf:
+            - $ref: '#/components/schemas/IdentificationTechnology'
+        connectivity_methods:
```
Comment (BenjaminPelletier) [inline interfaces/automated-testing/scd/scd.yaml:476]: Same comment as for `operation_mode`, but I don't think an "array of strings" is a very useful data type to define separately; I'd suggest we just define the data type of this field in the field itself:

```yaml
connectivity_methods:
  type: array
  items:
    type: string
  example: ['http://ussp.com/emergency']
```

File: interfaces/automated-testing/scd/scd.yaml [hunk 8]
```diff
@@ -36,6 +36,510 @@ components:
...
+          type: array
+          $ref: '#/components/schemas/ArrayOfConnectivityMethods'
+        endurance_minutes: 
+          type: integer
```
Comment (BenjaminPelletier) [inline interfaces/automated-testing/scd/scd.yaml:480]: Why not `number` here to be more flexible?

File: interfaces/automated-testing/scd/scd.yaml (hunk 1 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:450]: Thank you I have now fixed the name and also using the European spelling

File: interfaces/automated-testing/scd/scd.yaml (hunk 2 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:452]: Thank you, I have now fixed it.

File: interfaces/automated-testing/scd/scd.yaml (hunk 3 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:459]: Thank you, this is now fixed.

File: interfaces/automated-testing/scd/scd.yaml (hunk 4 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:467]: :) thanks, removed the part re the regulation reference

File: interfaces/automated-testing/scd/scd.yaml (hunk 5 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:468]: Thank you, I have now fixed it.

File: interfaces/automated-testing/scd/scd.yaml (hunk 6 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:472]: Thanks, I have now fixed it

File: interfaces/automated-testing/scd/scd.yaml (hunk 8 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:480]: Thank you, I have now made it a number.

File: interfaces/automated-testing/scd/scd.yaml (hunk 7 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:476]: OK understood, I have now fixed it.

File: interfaces/automated-testing/scd/scd.yaml [hunk 9]
```diff
@@ -36,6 +36,506 @@ components:
...
+          description: |-
+            Specify if the flight is "special" or has a specific priority.
+        uas_class: 
+          $ref: '#/components/schemas/UASClass'
+        identification_technologies: 
+          type: array
+          items:
+            type: string
+            enum:
+              - network
```
Comment (BenjaminPelletier) [inline interfaces/automated-testing/scd/scd.yaml:473]: Best practice for most enums is to have "Undeclared" or "Unknown" as the first option because some code generation will translate enums to integers and unspecified values become 0.  Also, to align with ASTM API style, let's capitalize the first letter of each value.

File: interfaces/automated-testing/scd/scd.yaml [hunk 10]
```diff
@@ -36,6 +36,506 @@ components:
...
+              - broadcast
+            description: Specify an identification technology that will be used for this flight
+        connectivity_methods:
+          type: array
+          items:
+            type: string
+          example: ['http://ussp.com/emergency']
```
Comment (BenjaminPelletier) [inline interfaces/automated-testing/scd/scd.yaml:480]: Is this a good example of a connectivity method?  It seems like maybe this is intended as an `emergency_procedure_url` example?

File: interfaces/automated-testing/scd/scd.yaml [hunk 11]
```diff
@@ -36,6 +36,506 @@ components:
...
+      - uas_serial_number
+      - operation_mode
+      - uas_class
+      - identification_technologies
+      - operator_id
+
+    OperationMode:
+      type: string
+      enum:
+        - vlos
```
Comment (BenjaminPelletier) [inline interfaces/automated-testing/scd/scd.yaml:502]: Same enum comment as above ("Unknown" or "Undeclared" as first entry, capitalize first letter)

File: interfaces/automated-testing/scd/scd.yaml [hunk 12]
```diff
@@ -36,6 +36,506 @@ components:
...
+      type: string
+      enum:
+        - C0
+        - C1
+        - C2
+        - C3
+        - C4
+      description: Specify the class of the UAS to be flown, the specifition matches EASA class identification label categories, for more information see EASA website
+
+    IdentificationTechnology:
```
Comment (BenjaminPelletier) [inline interfaces/automated-testing/scd/scd.yaml:516]: This should either be used with a `$ref` in the `items` of `identification_technologies` or removed

File: interfaces/automated-testing/scd/scd.yaml [hunk 13]
```diff
@@ -36,6 +36,506 @@ components:
...
+      type: string
+      enum:
+        - network
+        - broadcast
+      description: Specify an identification technology that will be used for this flight
+    
+    InjectFlightRequest:
+      type: object      
+      properties: 
+        injection_id:
```
Comment (BenjaminPelletier) [inline interfaces/automated-testing/scd/scd.yaml:526]: This PR will need to be rebased on #644 before it will be fully ready for review

File: interfaces/automated-testing/scd/scd.yaml [hunk 14]
```diff
@@ -36,6 +36,506 @@ components:
...
+        injection_id: 
+          description: 'ID of this injection.  Remains the same regardless of the flight ID reported in the system.'
+          type: string
+          example: 'edb7695f-8737-4b9f-91f8-e2afbb333f41'
+
+        operational_intent:
+          description: A Operational Intent for a flight provided for the test
+          $ref: '#/components/schemas/OperationalIntentTestInjection'
+
+        flight_authorization:
```
Comment (BenjaminPelletier) [inline interfaces/automated-testing/scd/scd.yaml:535]: nit: Probably want consistent spelling between field and data type

File: interfaces/automated-testing/scd/scd.yaml (hunk 9 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:473]: Thank you, I have now updated to add `undeclared` and capitalized the other two.

File: interfaces/automated-testing/scd/scd.yaml (hunk 11 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:502]: Thanks yes , I have now added Undeclared, and capitalized the rest of the items.

File: interfaces/automated-testing/scd/scd.yaml (hunk 12 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:516]: I have now removed this.

File: interfaces/automated-testing/scd/scd.yaml (hunk 14 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:535]: Thanks yes this is fixed now.

File: interfaces/automated-testing/scd/scd.yaml (hunk 10 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:480]: Yes you are correct, I have put in `radio` at the moment

File: interfaces/automated-testing/scd/scd.yaml (hunk 13 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:497]: I have now re-based the branch

**Comment by hrishiballal** [comment]
Rebased second time, based on @barroco 's comments to remove unused definitions

File: interfaces/automated-testing/scd/scd.yaml [hunk 15]
```diff
@@ -35,7 +35,478 @@ components:
...
+        emergency_procedure_url:
+          type: string
+          format: url
+          description: A valid url to contact in case of emergencies
+        operator_id:
+          type: string
+          example: "CHE0n4fujx7nwanb-fwi"
+          description: ID assigned to the operator per the EN4709-02 standard
+      required:
+      - uas_serial_number
```
Comment (barroco) [inline interfaces/automated-testing/scd/scd.yaml:363]: The required fields are the following:
- uas_serial_number
- operation_mode
- operation_category
- uas_class
- identification_technologies
- connectivity_methods
- endurance_minutes
- emergency_procedure_url
- operator_id

File: interfaces/automated-testing/scd/scd.yaml [hunk 16]
```diff
@@ -35,7 +35,478 @@ components:
...
+          description: 'ID of this injection.  Remains the same regardless of the flight ID reported in the system.'
+          type: string
+          example: 'edb7695f-8737-4b9f-91f8-e2afbb333f41'
+
+        operational_intent:
+          description: A Operational Intent for a flight provided for the test
+          $ref: '#/components/schemas/OperationalIntentTestInjection'
 
+        flight_authorisation:
+          description: A set of data about the operator to authorize a flight
```
Comment (barroco) [inline interfaces/automated-testing/scd/scd.yaml:398]: May I suggest to update the description like this:
```
Full description of a flight authorisation including mandatory information required by ANNEX IV of COMMISSION
        IMPLEMENTING REGULATION (EU) 2021/664 for an UAS flight authorisation request.
        Reference: https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32021R0664&from=EN#d1e32-178-1
```

File: interfaces/automated-testing/scd/scd.yaml [hunk 17]
```diff
@@ -35,7 +35,478 @@ components:
...
+          example: "normal"
+          description: |-
+            Specify if the flight is "special" or has a specific priority.
+        uas_class: 
+          $ref: '#/components/schemas/UASClass'
+        identification_technologies: 
+          type: array
+          items:
+            type: string
+            enum:
```
Comment (barroco) [inline interfaces/automated-testing/scd/scd.yaml:341]: Since those fields have not been specified/decided yet and we can't refer to any standard, I would recommend to leave it open as an array of string and remove the enum. We can still provide examples though.

File: interfaces/automated-testing/scd/scd.yaml [hunk 18]
```diff
@@ -35,7 +35,478 @@ components:
...
+              - Undeclared
+              - Network
+              - Broadcast
+            description: Specify an identification technology that will be used for this flight
+        connectivity_methods:
```
Comment (barroco) [inline interfaces/automated-testing/scd/scd.yaml:346]: Suggestion of description: 
```
            Connectivity methods.
            Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 7.
```

File: interfaces/automated-testing/scd/scd.yaml [hunk 19]
```diff
@@ -35,7 +35,478 @@ components:
...
+          type: array
+          items:
+            type: string
+          example: ['radio']
+        endurance_minutes: 
+          type: number
+          description: A number to specify the endurance of the drone in minutes
```
Comment (barroco) [inline interfaces/automated-testing/scd/scd.yaml:353]: Small description suggestion:
```
            Endurance of the UAS. This is expressed in minutes.
            Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 8.
```

File: interfaces/automated-testing/scd/scd.yaml [hunk 20]
```diff
@@ -35,7 +35,478 @@ components:
...
+        emergency_procedure_url:
+          type: string
+          format: url
+          description: A valid url to contact in case of emergencies
```
Comment (barroco) [inline interfaces/automated-testing/scd/scd.yaml:357]: Editoral suggestion:
```
            The URL at which the applicable emergency procedure in case of a loss of command and
            control link may be retrieved.
            Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 9.
```

**Review by barroco — CHANGES_REQUESTED** [review]
Thank you Hrishi for the PR. I made few suggestions to add references to the regulation.

File: interfaces/automated-testing/scd/scd.yaml [hunk 21]
```diff
@@ -35,7 +35,478 @@ components:
...
+
+    UASClass:
+      type: string
+      enum:
+        - C0
+        - C1
+        - C2
+        - C3
+        - C4
+      description: Specify the class of the UAS to be flown, the specifition matches EASA class identification label categories, for more information see EASA website
```
[the remark this answers is not in the document]
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:385]: Thank you for this, I have fixed the typo.

File: interfaces/automated-testing/scd/scd.yaml [hunk 22]
```diff
@@ -35,7 +35,478 @@ components:
...
+      description: Specify if the operation is a `vlos` or `bvlos` operation
+
+    UASClass:
+      type: string
+      enum:
+        - C0
+        - C1
+        - C2
+        - C3
+        - C4
```
[the remark this answers is not in the document]
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:384]: Thanks added this now.

[the remark this answers is not in the document]
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:384]: I have also added C5 and C6 and Other (at the top)

File: interfaces/automated-testing/scd/scd.yaml [hunk 23]
```diff
@@ -35,7 +35,478 @@ components:
...
+      - identification_technologies
+      - operator_id
+
+    OperationMode:
+      type: string
+      enum:
+        - Undeclared
+        - Vlos
+        - Bvlos
+      description: Specify if the operation is a `vlos` or `bvlos` operation
```
[the remark this answers is not in the document]
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:375]: Thanks added this now.

File: interfaces/automated-testing/scd/scd.yaml [hunk 24]
```diff
@@ -35,7 +35,478 @@ components:
...
+        A dataset to hold details of a UAS flight authorization request matching some of the requirements of Annex IV of the U-Space regulation 2021/664
+      type: object
+      properties:
+        uas_serial_number:
+          type: string
+          example: "T4S6BBMSZV4H0650"
+          description: Serial number of the UAS per the CTA-2063-A standard
+        operation_mode:   
+          $ref: '#/components/schemas/OperationMode'
+        operation_category:
```
[the remark this answers is not in the document]
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:330]: Thanks I have added it now and removed the `priority` reference.

File: interfaces/automated-testing/scd/scd.yaml (hunk 20 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:357]: Thanks! I have now modified this.

File: interfaces/automated-testing/scd/scd.yaml (hunk 19 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:353]: Thanks, I have now added this to the YAML

File: interfaces/automated-testing/scd/scd.yaml (hunk 18 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:346]: Thanks, I have now added this.

File: interfaces/automated-testing/scd/scd.yaml (hunk 16 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:398]: Thanks I have now added this to the YAML

File: interfaces/automated-testing/scd/scd.yaml [hunk 25]
```diff
@@ -35,7 +35,478 @@ components:
...
+          type: string
+          example: "normal"
+          description: |-
+            Specify if the flight is "special" or has a specific priority.
```
[the remark this answers is not in the document]
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:334]: OK I have now made the changes so that the three categories are reflected and the text updated

File: interfaces/automated-testing/scd/scd.yaml (hunk 15 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:363]: OK I have now made the fields mandatory, basically when we did the manual checkout we decided to test certain things but not others.

File: interfaces/automated-testing/scd/scd.yaml (hunk 16 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:398]: I have done this in the definition now.

File: interfaces/automated-testing/scd/scd.yaml [hunk 26]
```diff
@@ -35,7 +35,478 @@ components:
...
+          description: A number to specify the endurance of the drone in minutes
+        emergency_procedure_url:
+          type: string
+          format: url
+          description: A valid url to contact in case of emergencies
+        operator_id:
+          type: string
+          example: "CHE0n4fujx7nwanb-fwi"
+          description: ID assigned to the operator per the EN4709-02 standard
+      required:
```
[the remark this answers is not in the document]
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:363]: OK I have now added these links, but not made it as `required` I hope it is OK.

File: interfaces/automated-testing/scd/scd.yaml [hunk 27]
```diff
@@ -35,7 +35,478 @@ components:
...
+            $ref: '#/components/schemas/Volume4D'
+
+    FlightAuthorisationData:
+      description: |-
+        A dataset to hold details of a UAS flight authorization request matching some of the requirements of Annex IV of the U-Space regulation 2021/664
+      type: object
+      properties:
+        uas_serial_number:
+          type: string
+          example: "T4S6BBMSZV4H0650"
```
[the remark this answers is not in the document]
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:325]: Yes, for some reason VS Studio Code decided to revert my config. I have now reformatted it.

File: interfaces/automated-testing/scd/scd.yaml [hunk 28]
```diff
@@ -35,7 +35,373 @@ components:
...
+          type: number
+          description: Endurance of the UAS. This is expressed in minutes. Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 8.
+        emergency_procedure_url:
+          type: string
+          format: url
+          description: The URL at which the applicable emergency procedure in case of a loss of command and control link may be retrieved. Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 9.
+        operator_id:
+          type: string
+          example: "CHE0n4fujx7nwanb-fwi"
+          description: ID assigned to the operator per the EN4709-02 standard
```
Comment (BenjaminPelletier) [inline interfaces/automated-testing/scd/scd.yaml]: Could we add the [other information about this field](https://github.com/swiss-foca/Protocol/blob/061cb0fd0294818b9d02edb7865e05b4f31ed2b9/utm.yaml#L1149)?  In general, we have a pretty good prototype API for this information -- let's make sure we've taken full advantage of that head start.

File: interfaces/automated-testing/scd/scd.yaml [hunk 29]
```diff
@@ -35,7 +35,373 @@ components:
...
+      - endurance_minutes
+      - emergency_procedure_url
+
+    OperationMode:
+      type: string
+      enum:
+        - Undeclared
+        - Vlos
+        - Bvlos
+      description: Specify if the operation is a `vlos` or `bvlos` operation. Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 2.
```
Comment (BenjaminPelletier) [inline interfaces/automated-testing/scd/scd.yaml:399]: nit: In plain English, VLOS and BVLOS should be all caps because it is an acronym (visual line of sight).  (How they are in the enum values [looks fine](https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined) to me)

File: interfaces/automated-testing/scd/scd.yaml (hunk 28 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml]: Thanks, this is now updated.

File: interfaces/automated-testing/scd/scd.yaml (hunk 29 above)
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml:399]: I have made it all caps now. The `Undeclared` is kept as is.

File: interfaces/automated-testing/scd/scd.yaml [hunk 30]
```diff
@@ -35,7 +35,373 @@ components:
...
+          description: |-
+            Category of UAS operation (‘open’, ‘specific’, ‘certified’) as defined in COMMISSION DELEGATED REGULATION (EU) 2019/945. Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 4.
+        uas_class: 
+          $ref: '#/components/schemas/UASClass'
+        identification_technologies: 
+          type: array
+          items:
+            type: string
+            description: Specify an identification technology that will be used for this flight
+            example: ['Broadcast','Network']
```
[the remark this answers is not in the document]
Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml]: Thanks I have now moved the example indent one level up and used the specific `ADS-B` and `ASTMNetRID` here as well.

File: interfaces/automated-testing/scd/scd.yaml [hunk 31]
```diff
@@ -35,13 +35,398 @@ components:
...
+          description: |-
+            Category of UAS operation (‘open’, ‘specific’, ‘certified’) as defined in COMMISSION DELEGATED REGULATION (EU) 2019/945. Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 4.
+        uas_class:
+          $ref: '#/components/schemas/UASClass'
+        identification_technologies:
+          type: array
+          items:
+            type: string            
+          example: ['ADS-B', 'ASTMNetRID']
+          description: Specify an identification technology that will be used for this flight
```
Comment (barroco) [inline interfaces/automated-testing/scd/scd.yaml:345]: Editorial suggestion:
```
            Technology used to identify the UAS.
            Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 6.
```

Comment (hrishiballal) [reply interfaces/automated-testing/scd/scd.yaml]: Thanks! I have now added the examples and the definitions

**Comment by BenjaminPelletier** [comment]
This PR now needs to be rebased on master or have master merged into it to resolve conflicts.

File: interfaces/automated-testing/scd/scd.yaml [hunk 32]
```diff
@@ -309,16 +307,114 @@ components:
...
+        connectivity_methods:
+          type: array
+          items:
+            type: string
+          description: Connectivity methods. Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 7.
+          example: ['cellular']
+        endurance_minutes:
+          description: Endurance of the UAS. This is expressed in minutes. Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 8.
+          type: number
+          format: int32
```
Comment (BenjaminPelletier) [inline interfaces/automated-testing/scd/scd.yaml:352]: Adding this format defeats the purpose of [changing to number](https://github.com/interuss/dss/pull/645#discussion_r764351088), but I think we can remove this format later if we actually do need the flexibility a non-integer number provides.

**Review by BenjaminPelletier — APPROVED** [review]
